main/webapp/modules/core/langs/translation-en.json

```diff
@@ -309,7 +309,7 @@
     "core-facets/sort-by": "Sort by",
     "core-facets/name": "name",
     "core-facets/count": "count",
-    "core-facets/cluster": "Cluster",
+    "core-facets/cluster": "cluster",
```

Comment:
I think this one should stay capitalized, no? Before your change it looks like this, which I think is good:
![image](https://user-images.githubusercontent.com/309908/121502698-b4510d00-c9e0-11eb-9884-926d65007a93.png)